h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -75,6 +76,7 @@
   private final boolean enableGLRMReconstruct;  // if set true, will return the GLRM resconstructed value, A_hat=X*Y instead of just X
   private final boolean enableStagedProbabilities; // if set true, staged probabilities from tree agos are returned
   private final boolean enableContributions; // if set to true, will return prediction contributions (SHAP values) - for GBM & XGBoost
+  private final int glrmIterNumber; // allow user to set GLRM mojo iteration number in constructing x.
```

Comment:
do we really want to leak algo-specific configs here? wouldnt it be better to put into a subclass specific for glmr?

just thinking out loud here, but GlrmMojoModel could be the place for it

Reply:
GLRM is a little tricky.  It decomposes a dataset A = X*Y.  Sometimes the user just want X, sometimes the user want the reconstructed dataset from X*Y.  So, there are a lot of specific parameter for it.  Adding one more glrmIterNumber is yet another one.

Since we try to move all the scoring to mojo model directed by master Jedis (Michals), we will need to put these algo specific parameters somewhere.  Hence, in the mojo in this case.

Reply:
@wendycwong I think @honzasterba has a good point and I think you also have a good point that GLRM is very specific

it would be better to do it in a more generic way - please let me think about it